production/helm/loki/values.yaml

```diff
@@ -1178,13 +1178,17 @@ gateway:
       {{- end }}
     # -- Existing basic auth secret to use. Must contain '.htpasswd'
     existingSecret: null
+  # -- liveness probe for the nginx container in the gateway pods.
+  livenessProbe: {}
```

Comment:
What did you think about set

```yaml
  livenessProbe:
    httpGet:
      path: /
      port: http-metrics
    initialDelaySeconds: 15
    timeoutSeconds: 1
```

as well?

Reply:
My intention was to create a PR that makes the probes configurable but do not change any behavior, so it does not become a breaking change and have to wait until the next major release to be merged.

But if you prefer to also configure sensible probes wherever possible, that's fine by be. Do you want me to add the suggested probes?